editor/src/components/canvas/canvas-utils.ts

```diff
@@ -1748,16 +1751,18 @@ function getValidElementPathsFromElement(
   uiFilePath: string,
   isOnlyChildOfScene: boolean,
   parentIsInstance: boolean,
+  includeElementInPath: boolean,
```

Comment:
boo for one undecypherable more boolean flag

Reply:
Yeah, that should be changed.